cli/src/commands/branches/list.ts

```diff
@@ -1,15 +1,42 @@
-import { Command } from '@oclif/core';
+import { Flags } from '@oclif/core';
+import { BaseCommand } from '../../base.js';
+import { parseDatabaseURL } from '../../defaults.js';
 
-export default class BranchesList extends Command {
+export default class BranchesList extends BaseCommand {
   static description = 'List branches';
 
   static examples = [];
 
-  static flags = {};
+  static flags = {
+    workspace: Flags.string({
+      description: 'Workspace id to list branches from'
+    }),
+    database: Flags.string({
+      description: 'Database name to list branches from'
+    })
+  };
 
   static args = [];
 
-  async run(): Promise<void> {
-    this.error('To be done');
+  static enableJsonFlag = true;
+
+  async run(): Promise<any> {
```

Comment:
Not 100% about having ``any`` with the command output (applies to the rest of commands too.

I think we should have something as ``Branch[] | undefined``. We could even add a helper type ``Maybe<Branch[]>``.

If we won't never call the commands classes directly outside of testing (I think some commands will benefit from composition), maybe we can just remove the return type and let TS infer it.

Reply:
Sounds good 👍

Reply:
I think I'll do it in another PR.